readthedocs/donate/admin.py

```diff
@@ -1,8 +1,20 @@
 from django.contrib import admin
-from .models import (Supporter, SupporterPromo,
+from .models import (Supporter, SupporterPromo, Country,
                      PromoImpressions, GeoFilter)
 
 
+DEFAULT_EXCLUDES = ['BD', 'CN', 'TF', 'GT', 'IN', 'ID',
+                    'IR', 'PK', 'PH', 'RU', 'TW', 'TH', 'TR', 'UA', 'VN']
+
+
+def set_default_countries(modeladmin, request, queryset):
+    for project in queryset:
+        filter = project.geo_filters.create(filter_type='exclude')
+        for country in Country.objects.filter(country__in=DEFAULT_EXCLUDES):
+            filter.countries.add(country)
+set_default_countries.short_description = "Add default exclude countries to this Promo"
```

Comment:
This could belong in the `GeoFilterAdmin` class below, keeping everything encapsulated in the class.